### ISSUE TITLE
Current healthcare icon color too strong

### ISSUE THREAD

**Issue by matthijsmelissen** [gravitystorm/openstreetmap-carto#3613]
I think the current healthcare icon is too strong compared to other colors on the map. Would it be possible to turn down the saturation a bit?

<img width="425" alt="screen shot 2018-12-31 at 10 52 19" src="https://user-images.githubusercontent.com/5251909/50557946-2ee2b200-0cea-11e9-928d-2ebed21ba5c6.png">

CC @jragusa

**Comment by jeisenbe** [comment]
> turn down the saturation a bit

I agree, after comparing the new health color to  the other colors used for icons. It's the color with the highest chroma (saturation, "c") currently

_Ton line Left to right: gastronomy, **health**, shop, air-transport_:
![almost-all-icon-colors-current](https://user-images.githubusercontent.com/42757252/50966507-794d0580-1518-11e9-98ae-8f7473d21b02.png)
- Second row left to right: office, transportation, marina-text (fountains), water-text (waterfalls)
- Third row: amenity-brown, landform-color, man-made-icon, leisure (park darkened 60%)
- These are almost all of the current icon colors, except for black, used for religious features, and the blue used for springs, which is similar to water-color

I'd suggest reducing the chroma slightly to 72, which would be the same chroma as `@shop` - this would be `#b52613` or something similar.

It also looks like we should reduce the number of shades of blue.

**Comment by seav** [comment]
Aside from the color, I find it weird that the hospital and clinic icon starts rendering at zoom level 15, earlier than any other POI icon.

**Comment by kocio-pl** [comment]
Only hospital is drawn from z15 and it does not surprise me, since it is usually a big entity.

https://github.com/gravitystorm/openstreetmap-carto/blob/762b292b7ab9d353ed1fd99adabe6c44070977d4/amenity-points.mss#L654-L655

**Comment by seav** [comment]
![openstreetmap_-_2019-01-12_02 04 44](https://user-images.githubusercontent.com/873653/51051411-a8de3980-160e-11e9-8d8d-e229174a61d2.png)

Yes, they are usually big but not always. (In my hometown, most hospitals are actually small office-like buildings a few storeys high.) As such, they look really out of place on the map because they are the only large POIs that have icons at z15 and they stand out even more because of the saturated color. Note that other large POIs like colleges and universities don't even have icons.

**Comment by kocio-pl** [comment]
I would look closer on tagging, because from my experience many hospitals are de facto clinics, doctor's offices, hospital departments etc.

**Comment by seav** [comment]
If they are just doctor's offices, then they are tagged as clinics. But the hospitals here have emergency departments and rooms/wards for admitted patients.

**Comment by Adamant36** [comment]
@seav do you have a link to the example? Its usually helpful to provide one.

**Comment by seav** [comment]
Okay, here are three of several "small" hospitals within 3 kilometers of my residence:

1. Christ the King Medical Center - Unihealth Las Piñas
   - OSM: https://www.openstreetmap.org/node/414037437
   - `emergency=yes` + `beds=20`
2. Pope John Paul II Hospital and Medical Center
   - OSM: https://www.openstreetmap.org/way/363541233
   - `emergency=yes` + `beds=*` is unknown (newly opened hospital)
3. Las Piñas City Medical Center
   - OSM: https://www.openstreetmap.org/way/172909311
   - `emergency=yes` + `beds=66`

**Comment by kocio-pl** [comment]
When looking more on them, I guess rendering hospital icons from z16 would be OK for me too, because there are more city services like police or fire service. I feel no urge to change it however, for me this is open question and it depends on what others do think about it.

**Comment by jeisenbe** [comment]
I think we should try changing the color first.

We could also try using a smaller version of the icon on z15. It would be
legible at 10 by 10 pixel size.

The HDM/Humanitarian style does this for most icons.

**Comment by kocio-pl** [comment]
Hospitals really stand out at 15 and on z16 they're more balanced by other icons:

https://www.openstreetmap.org/#map=15/52.2211/20.9800

![screenshot_2019-01-16 openstreetmap](https://user-images.githubusercontent.com/5439713/51278269-73549a00-19da-11e9-823e-d10bc6793bc1.png)

![screenshot_2019-01-16 openstreetmap 1](https://user-images.githubusercontent.com/5439713/51278272-751e5d80-19da-11e9-8ad6-4e1ba2da05bf.png)

**Comment by jeisenbe** [comment]
@Tomasz-W do you think you could make a smaller hospital icon to be used at
z15? It could have a cross that is the same size as that in the pharmacy or
clinic icon, within a smaller circle. Perhaps 10px by 10px would work?

**Comment by Adamant36** [comment]
I thought there was a suggestion somewhere else recently to use multiple icons at different zoom levels for the same feature and it was shot down due to being over complication. 

To me doing that seems more like putting a bandaid on the problem then fixing it. The icon and color should be good enought to work at every zoom level or it should be changed. Plus if you do it here then it could lead to it being done in a lot of other places. Which would lead to massive icon bloat and a much more abstract map. Something people want to avoid. If you can't add new icons for new features due to bloat, you shouldnt be able to add new icons for features that already have icons.

**Comment by jeisenbe** [comment]
The HDM/Humanitarian style used smaller icons at lower zoom levels for
almost all icons, even though that style is generally much simpler than
this style.

A smaller version of an existing icon for an existing feature is quite
different than a new icon for a previously-unrendered feature.
